Lib/pathlib.py

```diff
@@ -1099,7 +1101,7 @@ def cwd(cls):
         """Return a new path pointing to the current working directory
         (as returned by os.getcwd()).
         """
-        return cls(os.getcwd())
+        return cls(cls()._accessor.getcwd())
```

Comment:
This is a quirky way to implement it. If the accessor becomes tied to the class rather than specific instances, perhaps it should become a class-level attribute?

Reply:
It's very similar to how `home()` is implemented immediately below.

Agree generally that the accessor should be a class attribute and not an instance attribute.

Reply:
I've replaced `cls()` with `cls` in this method, and `home()` for consistency.

Reply:
Well, that didn't work at all! Looks like:

- `_accessor` is set in `_init()`, at least until #19342 lands
- `_flavour` is set in `__new__()`.

Reverting to previous patch (the one you reviewed).